### PR TITLE
luci-base: zh_CN: update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by sotux**
update Simplified Chinese translation for the master branch.

As many new entires have been added, please review.
The entry "Potential negation of: %s" is not translated as I don't know the exact meaning of it.
Thanks.

### CODE REVIEW

File: modules/luci-base/po/zh-cn/base.po [hunk 1]
```diff
@@ -5294,25 +5303,29 @@ msgid ""
...
+msgstr "sysupgrade 命令失败，代码 %d"
 
 #: modules/luci-mod-system/htdocs/luci-static/resources/view/system/flash.js:269
 msgid ""
 "The uploaded backup archive appears to be valid and contains the files "
 "listed below. Press \"Continue\" to restore the backup and reboot, or "
 "\"Cancel\" to abort the operation."
 msgstr ""
+"上传的备份归档有效，并且包含以下列出的文件。按“继续”恢复备份并重新启动，或"
+"按“取消”中止操作。"
```
Comment (yousong) [inline modules/luci-base/po/zh-cn/base.po]: “按”译为“点击”是不是更自然一些。“恢复备份”可考虑像上文“Apply backup”一样译为“应用备份”

File: modules/luci-base/po/zh-cn/base.po [hunk 2]
```diff
@@ -5348,18 +5361,20 @@ msgstr "中继的 IPv4 地址"
...
+msgstr "这不是有效的 PEM 文件"
 
 #: modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js:163
 msgid ""
 "This file may contain lines like 'server=/domain/1.2.3.4' or "
 "'server=1.2.3.4' for domain-specific or full upstream <abbr title=\"Domain "
 "Name System\">DNS</abbr> servers."
 msgstr ""
+"对于特定于域的或完整的上游 <abbr title =\"Domain Name System\"> DNS </abbr> "
+"服务器，此文件可能包含“server=/domain/1.2.3.4”或“server=1.2.3.4”之类的行。"
```
Comment (yousong) [inline modules/luci-base/po/zh-cn/base.po]: “此文件可能包含格式如 xxx、yyy 的行，前者为特定的域指定DNS服务器，后者则不限定服务器的解析范围。”

File: modules/luci-base/po/zh-cn/base.po [hunk 3]
```diff
@@ -6061,7 +6078,7 @@ msgstr "是"
 msgid ""
 "You appear to be currently connected to the device via the \"%h\" interface. "
 "Do you really want to shut down the interface?"
-msgstr ""
+msgstr "您似乎当前已通过“%h”接口连接到设备。您真的要关闭接口吗？"
```
Comment (yousong) [inline modules/luci-base/po/zh-cn/base.po]: “您似乎正通过 "%h" 连接到此设备，确认要关闭它吗”

**Comment by sotux** [comment]
Thank you @yousong , I've updated the translation.

**Comment by feckert** [comment]
@yousong Now do you agree with the changes?

**Comment by yousong** [comment]
Thanks for the efforts!

**Pull request merged**